### ISSUE TITLE
Every(!) touch to graphs lead to detail value display even if not intended

### ISSUE THREAD

**Issue by jubie25** [home-assistant/frontend#18198]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Coming from 
https://github.com/home-assistant/android/issues/3932#issuecomment-1759514562

Concerns the use of the GUI on Android devices: Home Assistant app and browsers, here: Chrome, Fenec (Firefox). Here:
Home Assistant Android app version 2023.9.2-full (and older)

*Every* touch on the display targeting a graph leads to instantly display an overlay window showing detail values to that point. This is true even if the touch is a swipe gesture. The popping up overlay window is rather persistent. Only a touch again within the graph makes is go away (and only if placed wise). This limits daily user experience and lead to confusing unexperienced users (wall mounted display)
![274559781-4f1ea008-0ca1-4632-9b71-d1f06c5f1b0b](https://github.com/home-assistant/frontend/assets/72253640/9b9b5240-220e-486f-ac11-c3cd3ab97bf1)

### Describe the behavior you expected

A proposal could be:
- Detailed overlay infos should come up only after long press.
- Maybe a switch to totally disable those overlays makes sense (for wall mounted info displays).

### Steps to reproduce the issue

1. use App or browser on Android
2. swipe the display up/down, touching it within a graph near some data points
3. an overlay appears immediately showing detail info. Hard to get rid of it

### What version of Home Assistant Core has the issue?

2023.10.1. and older

### What was the last working version of Home Assistant Core?

never?

### In which browser are you experiencing the issue with?

Chrome 117.0.5938.153 and older. Fenec/Firefox 118.0.0

### Which operating system are you using to run this browser?

Android 14 / Android 13

### State of relevant entities

```txt
not applicable
```

### Problem-relevant frontend configuration

```yaml
not applicable
```

### Javascript errors shown in your browser console/inspector

```txt
not applicable
```

### Additional information

see above

**Comment by catalincre** [comment]
Me to.
![Screenshot_20231017_071733_Home Assistant](https://github.com/home-assistant/frontend/assets/122742384/2af7da86-5eec-4aea-9981-1398bd4a82e6)

**Comment by MrReSc** [comment]
Same here.

**Comment by i-jk** [comment]
Yes, this is making graphs and the app itself actually hard to use. I think this started in September. Having floating point details boxes accumulate with accidental touches, or bring taken off to the full history when I want a value is incredibly frustrating.

**Comment by i-jk** [comment]
I wonder if it's all related to this commit?: https://github.com/home-assistant/frontend/commit/dd6a69ea03e8f9b2ddc696e3b3b5290eb400abfb

**Comment by adamsderk** [comment]
Would be nice to have an option in the visual editor to run it on an off (after fixing it getting stuck on of course).

**Comment by k3a** [comment]
> I wonder if it's all related to this commit?: [dd6a69e](https://github.com/home-assistant/frontend/commit/dd6a69ea03e8f9b2ddc696e3b3b5290eb400abfb)
> 
> Lots of calls so to "more-info" added in here

I am the author of mentioned PR. If I understand it correctly this issue #18198 is about tooltips, not more-info. This problem with tooltips was there even before my PR, you can try ( `ghcr.io/home-assistant/home-assistant:2023.9.3` is the build before my PR, when trying make sure to backup your home-assistant config directory as downgrading will mess it up). 

I also had to deal with this tooltip problem in the followup PR by simulating `mouseout` on the chart canvas before displaying more-info https://github.com/home-assistant/frontend/pull/18103

The problem is that `mouseout` event is not reliably delivered to the canvas on mobile platforms for some reason. There is a relevant PR on the chart.js https://github.com/chartjs/Chart.js/issues/7738

**Comment by jubie25** [comment]
I'm no Android programmer, my skills are limited dealing with it. According to the timestamp in the build you mentioned this should show the problem, too, because the issue is observed by me for a longer time now.
You mentioned "mouseout". As said, I'm not familiar with Android but may it be possible to trigger the tooltip by a "long pressed mousein"? 
Many of the users of my home automation are "user-only family members" that don't care about details. Or they must not have the ability to dive into details/configuration and so on. So it also may be a (additional) solution to disable tooltips/details at all on a per-user or per-device base.

**Comment by i-jk** [comment]
@k3a thanks for commenting.

All I know is these charts became much less usable after your commits:  tooltips floating around all over the place, often blocking UI important elements in popups. This widget is much less usable now, I'm sorry to report.

**Comment by Thaijoni** [comment]
Still the same for me...

Skickat från Outlook för Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by del13r** [comment]
Hello, this is still an issue for me. It is really disappointing that no one is really owning this issue. The product was working fine in August 2023, and has been frustrating to operate on mobile since September/october 2023.

**Comment by del13r** [comment]
> This problem with tooltips was there even before my PR, you can try ( `ghcr.io/home-assistant/home-assistant:2023.9.3` is the build before my PR, when trying make sure to backup your home-assistant config directory as downgrading will mess it up).

I am actually going to try this today to work out exactly which version the problem started with.

**Comment by k3a** [comment]
The problem with tooltips over the side menu was attempted to be fixed by this commit (by setting CSS z-index to 1) https://github.com/home-assistant/frontend/commit/fb73bfb96402b1f03bcf8c09251c3b34195c0982

(tags [20240307.0](https://github.com/home-assistant/frontend/releases/tag/20240307.0) [20240306.0](https://github.com/home-assistant/frontend/releases/tag/20240306.0) [20240304.0](https://github.com/home-assistant/frontend/releases/tag/20240304.0) [20240301.0](https://github.com/home-assistant/frontend/releases/tag/20240301.0) [20240228.1](https://github.com/home-assistant/frontend/releases/tag/20240228.1) [20240228.0](https://github.com/home-assistant/frontend/releases/tag/20240228.0) [20240207.1](https://github.com/home-assistant/frontend/releases/tag/20240207.1) [20240207.0](https://github.com/home-assistant/frontend/releases/tag/20240207.0) [20240205.0](https://github.com/home-assistant/frontend/releases/tag/20240205.0) [20240202.0](https://github.com/home-assistant/frontend/releases/tag/20240202.0))

Is it really still a problem? What platform?

**Comment by del13r** [comment]
Hi @k3a,
I documented my issue under
https://github.com/home-assistant/frontend/issues/18670
Currently running 2024.3.1 and issue is still there.

**Comment by jubie25** [comment]
There has nothing changed. The issue still exists the same as before. tested with 2024.1.5-full. Please don't stale or drop this issue.

**Comment by jubie25** [comment]
This issue isn't resolved and it impairs daily use of Home Assistant on touch devices. 
Personal note: it lowers the "women acceptance factor" WAF by a significant value, leading to inacceptance of the system because of no-use... :-)

**Comment by jubie25** [comment]
I want to ask for resoving this issue. It's REALLY annoying. Yesterday I finally brought a wall terminal in "official" use. Those notification on a single tap are hiding the interesting parts, even if this tap was made to swipe/scroll the content. I earned a "NO GO" from my family members....

**Comment by jubie25** [comment]
Hello again, is there any life in this matter? I istalled several tablets as room HMI and get feedback of all those anoying popups when touching/scrolling/zooming a page. Please, I want to ask if this is a matter of change (and doesn't have ressources by now) or if nothing will happen here (maybe because all users are considered being tolerant admins, not simple family member users...)
Sorry, I'm a bit frustrated. Your work is the more great the more I dig into the system!

**Comment by jubie25** [comment]
I want to ask for any progress or intentions in this matter. This is a constant nuisance when using the frontend on touch based devices. I'm finding myself doing avoidance strategies when touching chart pages to scroll or zoom them. And this shouldn't be at all.
So please: is there any remedy planned? Are the developers aware of it? I wish I had the skills to help here but I'm "only" C programmer.

**Comment by nilsAx** [comment]
I also want to know if this is being worked on! It is incredibly annoying, to the point where I was searching for ways to disable the tooltip entirely - which is how I found this discussion.

**Comment by melyux** [comment]
Has gotten even worse since 2025.1 history graph update.

**Comment by jubie25** [comment]
ist there any life from the developers/maintainers? Ist this problem aware? I cannot believe we're the only one who install a home panel or want to control our home by touch instead of mouse.
Maybe we get a ping back even if there's no one at the moment who is able to fix it?

**Comment by iotola** [comment]
I agree this fault makes graphs almost unusable on android. Zooming has long been a problem and the advanced zooming in the 2025.01 version is awful due to this. I agree that long press is a good solution, alternatively it needs some sort of toggle to turn it on or off.

**Comment by del13r** [comment]
Things seem to be much better after 2025.2 release.
Note on what changed https://www.home-assistant.io/blog/2025/02/05/release-20252/#preparing-our-graphs-for-the-future

**Comment by jubie25** [comment]
I can confirm: things now are muuuuch better than before. With 2025.1 the behaviour reached it's lowest point because this new zooming feature broke any attempt to use the graphs at all on devices where one use HA by touch (Android/phone/tablet...).
With 2025.2 you now can scroll without being stopped by unintended detail popups. And even zooming with two-finger-gesture is fine.
I wondered all the time about the silence from the developers for this issue. Now we learned: they were working on a new backend for graphs. And so didn't spend time in fixing the old (i think). This would have been an information that may have helped us through this whole thing....

I think this issue could be closed now. The main goal is reached!

**Comment by del13r** [comment]
> I think this issue could be closed now. The main goal is reached!

I think you are the only person who can close it as you raised the issue.
